json/src/main/java/com/basistech/rosette/apimodel/jackson/AddressSimilarityRequestMixin.java

```diff
@@ -22,7 +22,7 @@
 import com.fasterxml.jackson.annotation.JsonTypeName;
 import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
 
-import com.basistech.rosette.apimodel.Address;
+import com.basistech.rosette.apimodel.IAddress;
```

Comment:
I don't think we want to rename a class in the Java binding already released once if avoided. How bad if you had to keep "Address" instead? Can we make sure existing clients compiled without modifications?

Reply:
`Address` is still around (though marked as deprecated), `IAddress` is simply a new interface it now implements.  I built the D&B project locally with a snapshot bindings release and everything compiled successfully; I think existing clients should just be able to drop in a new release without an issue